debian/control

```diff
@@ -21,7 +21,6 @@ Build-Depends: automake,
                golang,
                gperf,
                g++ (>= 7),
-               hostname <pkg.ceph.check>,
```

Comment:
what is this for?

Reply:
hostname is in the Essential set of packages, which nothing should declare a dependency upon - https://www.debian.org/doc/debian-policy/ch-binary.html#dependencies

Reply:
thank you. my fault. should have read the document more closely.